docs/user/connected-accounts.rst

```diff
@@ -1,28 +1,51 @@
-Connecting Your VCS Account
-===========================
+Connecting your Git repository
+==============================
+
+In this article, we explain how connecting your Read the Docs account to one of the supported Git providers
+automatically configures your Git repository and your Read the Docs project.
+
+✅️ Signed up with your Git provider?
+  If you signed up or logged in to Read the Docs with your GitHub, Bitbucket, or GitLab
+  credentials, you're all done. Your account is connected and the integration explained
+  below is enabled.
+⬇️ Signed up with your email address?
+  If you have signed up to Read the Docs with your email address,
+  you can add the connection to the Git provider afterwards.
+  Read more about how it works below.
+
 
 If you are going to import repositories from GitHub, Bitbucket, or GitLab,
-you should connect your Read the Docs account to your repository host first.
+we recommend that you connect your Read the Docs account to your repository provider.
+
 Connecting your account allows for:
 
-* Easier importing of your repositories
-* Automatically configure your repository :doc:`/integrations`
+* Easy import of your repositories
+* Automatic configuration of your repository :doc:`/integrations`
   which allow Read the Docs to build your docs on every change to your repository
-* Log into Read the Docs with your GitHub, Bitbucket, or GitLab credentials
+* Logging into Read the Docs with your GitHub, Bitbucket, or GitLab credentials
```

Comment:
These should all end with a period, and we should note that in the style guide :)

Reply:
Fixed :)

Quick eyes sometimes don't get old structures updated.